app/renderer/actions/Session.js

```diff
@@ -181,7 +181,8 @@ export function newSession (caps, attachSessId = null) {
         https = session.server.remote.ssl;
         break;
       case ServerTypes.sauce:
-        host = 'ondemand.saucelabs.com';
+        host = session.server.sauce.dataCenter === 'eu-central-1' ?
```

Comment:
I'd like it if `eu-central-1` and `us-west-1` weren't hardcode but rather defined as constants in this fie (actions/Session.js).